scripts/a11y/a11y-output-txt.txt

```diff
@@ -0,0 +1,2619 @@
+[ { violations: 
```

Comment:
I don't think we want to check these in

Reply:
Also, does puppeter off a way to display this output in a way that is easier to read?

Reply:
I made it so the output is printed as both a .txt file and .csv file and included it in the PR (see files changed).